addons/website_sale/static/src/scss/website_sale.scss

```
@@ -235,7 +235,7 @@ $o-wsale-products-layout-grid-gutter-width: min($grid-gutter-width / 2, $o-wsale
         padding: var(--o-wsale-card-info-padding, #{map-get($spacers, 2)} 0);
     }
 
-    [data-oe-field="description_sale"][data-oe-type] {
+    .oe_subdescription {
```

This isn't targetting exactly the same element, I would add a div to make sure it's the same
```suggestion
    .oe_subdescription div {
```